File: LEVEL1/두개뽑아서더하기.py
lst = [5, 0, 2, 7]
array = []
answer = []

for i in range(len(lst)):
    for j in range(len(lst)):
        if i == j:
            break
        else:
            array.append(lst[i] + lst[j])

answer = list(set(array))
answer.sort()

print(answer)

File: LEVEL1/Javascript/두개뽑아서더하기.js
function solution(numbers) {
  let tmp = [];
  
  for (let i = 0; i < numbers.length; i++) {
      for (let j = 0; j < numbers.length; j++) {
          if (i !== j) tmp[tmp.length] = numbers[i] + numbers[j];
      }
  }
  
  let answer = tmp.filter((e, i) => {
      return tmp.indexOf(e) === i;
  });

  answer.sort((a, b) => (a - b));
  
  return answer;
}